benchmarks/transport_montecarlo_packet.py

```diff
@@ -37,9 +35,7 @@ def geometry(self):
 
     @property
     def model(self):
```

Comment:
let's rename that property

Reply:
I think Asish can do this as part of his project

Reply:
is this an issue?